docs/modules/ROOT/pages/documentation/payara-server/app-deployment/deployment-descriptors.adoc

```diff
@@ -75,3 +75,30 @@ Since version __5.184__
 Since version __5.2020.5__
 
 * `log-jdbc-calls` (_attribute of jdbc-connection-pool_)
+
+
+[[payara-server-deployment-descriptor-precedence-order]]
+== Payara Server Deployment Descriptor Precedence Order
+
+When multiple deployment descriptors are present in the same archive, DDs are
+ processed with the following precedence order in default mode:
+
+    `WebLogic DD > Payara DD > GlassFish DD > Sun DD`
+
+The default precedence order can be customized and configured via following
+ System properties:
+
+[[gfdd.over.wlsdd]]
+==== `gfdd.over.wlsdd` property
+This system property controls the precedence between GlassFish Server 
+DD and Weblogic Server DD when they are both present. When `gfdd.over.wlsdd`
+ property is set to true, DDs are processed with the following precedence order:
+
+    `Payara DD > GlassFish DD > WebLogic DD > Sun DD`
+
+[[ignore.wlsdd]]
+==== `ignore.wlsdd` property
+When `ignore.wlsdd` property is set to true, WLS DD will be ignored and other
+ DDs will be processed with the following precedence order:
+
+    `GlassFish DD > Payara DD > Sun DD`
```

Comment:
I would replace DD with descriptor, it's still shorter than "deployment descriptor" but easier to read than DD.
```suggestion
When multiple deployment descriptors are present in the same archive, the descriptors are
 processed with the following precedence order in default mode:

    `WebLogic descriptor > Payara descriptor > GlassFish descriptor > Sun descriptor`

The default precedence order can be customized and configured via following
 system properties:

[[gfdd.over.wlsdd]]
==== `gfdd.over.wlsdd` property
This system property controls the precedence between GlassFish Server 
descriptor and Weblogic Server descriptor when they are both present. When `gfdd.over.wlsdd`
 property is set to true, the descriptors are processed with the following precedence order:

    `Payara descriptor > GlassFish descriptor > WebLogic descriptor > Sun descriptor`

[[ignore.wlsdd]]
==== `ignore.wlsdd` property
When `ignore.wlsdd` property is set to true, WLS descriptor will be ignored and other
 descriptors will be processed with the following precedence order:

    `GlassFish descriptor > Payara descriptor > Sun descriptor`
```